vendor/assets/stylesheets/ombulabs/styleguide/1-utilities/_breakpoints.scss

```diff
@@ -0,0 +1,105 @@
+//
+//  MEDIA QUERIES
+//––––––––––––––––––––––––––––––––––––––––––––––––––
+
+// A map of breakpoints.
+$breakpoints: (
+  xs: 576px,
+  sm: 768px,
+  md: 992px,
+  lg: 1200px,
+  xlg: 1600px
```

Comment:
@nykka Could you use real words? There is no need to make variable names that small. For example: "extra-small", "small", etc...